modules/passkey/contracts/test/TestWebAuthnLib.sol

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import {WebAuthn} from "../libraries/WebAuthn.sol";
+import {IP256Verifier} from "../libraries/P256.sol";
+
+contract TestWebAuthnLib {
+    /**
+     * @notice Generate a signing message based on the authenticator data, challenge, and client
+     * data fields.
+     * @dev The signing message are the 32-bytes that are actually signed by the P-256 private key
+     * when doing a WebAuthn credential assertion. Note that we verify that the challenge is indeed
+     * signed by using its value to compute the signing message on-chain.
+     * @param challenge The WebAuthn challenge used for the credential assertion.
+     * @param authenticatorData Authenticator data.
+     * @param clientDataFields Client data fields.
+     * @return message Signing message.
+     */
```

Comment:
nit: why natspec for test contracts? Feels like overkill, and can potentially get out of sync over time.

Reply:
Copy and paste from the lib code. Will remove